movieHW/MainModule/Views/MoviesViewController.swift

```diff
@@ -12,42 +12,80 @@ final class MoviesViewController: UIViewController {
         static let soonButtonText = "Скоро"
         static let popularButtonText = "Популярное"
         static let bestButtonText = "Лучшее"
-    }
-
-    private enum CellIdentifier {
+        static let fatallErrorText = "init(coder:) has not been implemented"
         static let movieCellIdentifier = "MovieTableViewCell"
-    }
-
-    private enum Colors {
         static let defaultOrange = "defaultOrange"
         static let defaultBlack = "defaultBlack"
     }
 
-    private enum UrlStrings {
-        static let popularUrlString =
-            "https://api.themoviedb.org/3/movie/popular?api_key=c4c9f43f1e9f5412ba0efa79f720438e&language=en-US&page=1"
-        static let topRatedUrlString =
-            "https://api.themoviedb.org/3/movie/top_rated?api_key=c4c9f43f1e9f5412ba0efa79f720438e&language=en-US&page=1"
-        static let soonUrlString =
-            "https://api.themoviedb.org/3/movie/upcoming?api_key=c4c9f43f1e9f5412ba0efa79f720438e&language=en-US&page=1"
-    }
-
     // MARK: Private Visual Components
 
-    private var movies: [Results] = [] {
+    private var movies: [Movie] = [] {
         didSet {
             DispatchQueue.main.async {
                 self.tableView.reloadData()
             }
         }
     }
 
-    private let request = MovieRequest()
     private let refresherControl = UIRefreshControl()
+
     private let tableView = UITableView()
-    private let popularButton = UIButton(type: .system)
-    private let topRatedButton = UIButton(type: .system)
-    private let comingSoonButton = UIButton(type: .system)
+
+    private let popularButton: UIButton = {
+        let popularButton = UIButton()
+        popularButton.setTitle(Constants.popularButtonText, for: .normal)
+        popularButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        popularButton.backgroundColor = UIColor(named: Constants.defaultOrange)
+        popularButton.setTitleColor(UIColor.white, for: .normal)
+        popularButton.layer.cornerRadius = 9
+        popularButton.layer.borderColor = UIColor.white.cgColor
+        popularButton.layer.borderWidth = 1.0
+
+        return popularButton
+    }()
+
+    private let topRatedButton: UIButton = {
+        var topRatedButton = UIButton()
+        topRatedButton.setTitle(Constants.bestButtonText, for: .normal)
+        topRatedButton.setTitleColor(UIColor.white, for: .normal)
+        topRatedButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        topRatedButton.backgroundColor = UIColor(named: Constants.defaultOrange)
+        topRatedButton.layer.cornerRadius = 9
+        topRatedButton.layer.borderColor = UIColor.white.cgColor
+        topRatedButton.layer.borderWidth = 1.0
+
+        return topRatedButton
+    }()
+
+    private let comingSoonButton: UIButton = {
```

Comment:
```suggestion
    private let comingSoonButton: UIButton = {
```
 создай 1 метод и передавай параметры туда, структура одинаковая у всех трех